README.md

```diff
@@ -86,6 +86,11 @@ To test, run
 bazel test //p4_contraints/...
 ```
 
+To see the output of a failed test, invoke it using `bazel run` like so:
+```sh
+bazel run //p4_constraints/frontend:lexer_test
```

Comment:
This seems a bit of a weird recommendation, wouldn't you just use standard bazel things like bazel test .. --test_output=streamed?

Reply:
Bazel (or at least blaze) does not build test targets in debug mode by default (presumably to speed up testing?), so if the program crashes you don't get useful stack traces unless you do
```sh
bazel test <target> --test_output=streamed --compilation_mode=dbg
```
That's a bit of a mouthful, so in practice I just use
```sh
bazel run <target>
```
which does the right thing.

Reply:
Aha. I'm fine either way, it still feels a bit like we are giving a bazel tutorial. But I'm happy to have it if you feel like it helps :)

Reply:
True, it is a Bazel tutorial. I'm just worried newcomers will get frustrated with Bazel right away without it since Bazel's defaults here are unfortunate and it's not as easy as you would hope to figure out what to do